packages/compiler-core/src/utils.ts

```diff
@@ -84,8 +84,10 @@ export const isSimpleIdentifier = (name: string): boolean =>
   !nonIdentifierRE.test(name)
 
 const memberExpRE = /^[A-Za-z_$][\w$]*(?:\.[A-Za-z_$][\w$]*|\[[^\]]+\])*$/
-export const isMemberExpression = (path: string): boolean =>
-  memberExpRE.test(path)
+export const isMemberExpression = (path: string): boolean => {
+  if (!path) return false
+  return memberExpRE.test(path.trim())
+}
```

Comment:
This could also be done by only regular expression
```regex
^[A-Za-z_$\s][\w$]*(?:\.[A-Za-z_$][\w$]*|\[[^\]]+\]|\s)*$
```

But I think using trim is a bit more explicit than changing the regex

Reply:
not support `v-model="obj[keys[name]]"`

Reply:
nor `getModel(key).value`